tests/SConscript

```diff
@@ -1,3 +1,9 @@
 # -*- python -*-
-from lsst.sconsUtils import scripts
+import os
+from lsst.sconsUtils import env, scripts
```

Comment:
Usually I am asked to leave a blank line between imports of standard modules and of non-standard imports which would mean that a blank line is required between these two imports. I'll leave it to you to decide whether you want to introduce one or not.